integrations/electricity/energinet.js

```diff
@@ -0,0 +1,184 @@
+import request from 'superagent';
```

Comment:
I think you don't need superagent - you could just use the natice fetch API. It would make for one less dependency. Not a big problem though.

Reply:
I used the orsted integration for reference, so I didn't really think about it